### PR TITLE
namespace for shortcode (#8)

### PR DESCRIPTION
**Pull request by ChristofLee**
Add namespace to shortcode and functions, checking the functions exist before defining them.

This PR also merges #1 and #2 so suggest merging those PR's first and then reviewing this.

### CODE REVIEW

**Comment by ChristofLee** [comment]
This renames shortcode functions to be more specific and checks if each shortcode function exists before defining it, preventing conflicts.

**Comment by warengonzaga** [comment]
I have pending requirements, can you check it @ChristofLee here in this issue #8 ?

File: shortcodes-wp.php [hunk 1]
```diff
@@ -15,66 +15,79 @@
...
 }
 
 // query param shortcode
-add_shortcode('wpuser_query_param', 'get_query_param');
-function get_query_param($atts){
-    $atts = shortcode_atts( array(
-        'arg' => false,
-    ), $atts);
-    return isset( $_GET[ $atts['arg'] ] ) ? sanitize_text_field( $_GET[ $atts['arg'] ] ) : false;
+add_shortcode('wpuser_query_param', 'wp_info_get_query_param_shortcode');
```
Comment (warengonzaga) [inline shortcodes-wp.php]: The ``wpuser_query_param`` should be ``wpinfo_query_param``

if the shortcode is related to generic WordPress information please use the shortcode guide here #8 

> Then for the shortcode please use the following.
> 
> ``[wpinfo_*]`` for info related to WP
> ``[wpuser_*]`` for info related to WP user.

**Review by warengonzaga — CHANGES_REQUESTED** [review]
Please review my comment for the query feature.

File: shortcodes-wp.php (hunk 1 above)
Comment (ChristofLee) [reply shortcodes-wp.php]: My mistake, I have updated this now.

**Review by warengonzaga — APPROVED** [review]
Looks good now!

**Comment by warengonzaga** [comment]
Thanks, @ChristofLee for the help! I'm merging this now to the ``dev`` branch.

**Pull request merged**